sw/device/silicon_creator/lib/cert/dice.c

```diff
@@ -86,9 +112,32 @@ static void curr_tbs_signature_le_to_be_convert(void) {
                    kAttestationSignatureBytes / 2);
 }
 
+/**
+ * Helper function to compute measurements of various OTP partitions that are to
+ * be included in attestation certificates.
+ */
+static status_t measure_otp_partition(otp_partition_t partition,
```

Comment:
Follow-up PR: we should not be using `status_t` in silicon_creator code.  We should be using rom_error_t.

Reply:
Yes, this is a refactoring I am currently doing.